sdk/cosmosdb/cosmos/test/functional/sproc.spec.ts

```diff
@@ -62,13 +62,13 @@ describe("NodeJS CRUD Tests", function() {
 
       // replace sproc
       // prettier-ignore
-      sproc.body = function () { const x = 20; };
+      sproc.body = function () { const x = 20; console.log(x); };
```

Comment:
Do you want these console.logs in the tests?

Reply:
Yeah. These are the bodies of what gets sent to the server. Had to add the logs so typescripts noUnusedLocals wouldn't get complain.

Reply:
You can also just reference it, e.g. `const x = 20; x;` should work.